collectors/etc/grok_exporter_conf.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+
+"""
+Config for the grok processes to be run
+"""
+
+GROK_EXPORTER_DIR = '/usr/local/grok_exporter-0.2.1.linux-amd64'
+
+CONFIG_FILES = [
+   'nginx.yml',
+   'tomcat.yml'
+]
```

Comment:
Will we be changing the existing Ansible task so that it changes this to include the opentsdb and tcollector ymls for the TSD nodes?